bin/test.sh

```diff
@@ -31,7 +31,7 @@ echo "Checking redis-server responds to ping command..."
 # Redis <4 does not support connection URLs so REDIS_URL has to be parsed:
 # https://stackoverflow.com/questions/38271281/can-i-use-redis-cli-with-a-connection-url
 REDIS_CONNECTION_ARGS="\$(echo \${REDIS_URL} | sed 's_redis://h:\(.*\)@\(.*\):\(.*\)/_-h \2 -p \3 -a \1_')"
-TEST_COMMAND="source .profile.d/redis.sh && redis-cli ${REDIS_CONNECTION_ARGS} ping"
+TEST_COMMAND="source .profile.d/redis.sh && redis-cli -r 5 -i 2 ${REDIS_CONNECTION_ARGS} ping"
```

Comment:
This doesn't work, it still fails on the first attempt. Makes sense when hindsight - this is for _repeating_ a command, not _retrying_. Instead I'll look at using something (`nc`?) to wait for the port to open. And if _that_ doesn't work then go back to the `sleep` 😄

Reply:
Couldn't see a clean way of doing it so sticking with the `sleep` for now.